PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -58,6 +58,8 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
             switch filter {
             case .favorites:
                 predicates.append(NSPredicate(format: "isFavorite = true", true))
+            case .all:
+                break
```

Comment:
I like the use of `compactMap` in `ArchivedItemsListViewModel`; can we repurpose that here?

Reply:
yes! also, curious to hear why we have the additional argument for `NSPredicate(format: "isFavorite = true", true)` vs `NSPredicate(format: "isFavorite = true")`

Reply:
@cyndichin I think it may just be an oversight; it should work fine without the extra arg.